README.md

```diff
@@ -4,6 +4,7 @@
 [![AppVeyor build status](https://img.shields.io/appveyor/build/Tieske/penlight-ta1gi/master?logo=appveyor)](https://ci.appveyor.com/project/Tieske/penlight-ta1gi/branch/master)
 [![Coveralls code coverage](https://img.shields.io/coveralls/github/lunarmodules/Penlight?logo=coveralls)](https://coveralls.io/github/lunarmodules/Penlight)
 [![Luacheck](https://github.com/lunarmodules/Penlight/workflows/Luacheck/badge.svg)](https://github.com/lunarmodules/Penlight/actions)
+[![SemVer](https://img.shields.io/badge/SemVer-1.9.2-brightgreen?logo=semver)](CHANGELOG.md)
```

Comment:
```suggestion
[![SemVer](https://img.shields.io/github/v/tag/lunarmodules/Penlight?color=brightgreen&label=semver&sort=semver)](CHANGELOG.md)
```

Here is a badge URL that wouldn't need updating on every release, it will show the latest tag.

Reply:
nice! didn't know that one